.github/workflows/publish_api.yaml

```diff
@@ -83,6 +83,12 @@ jobs:
       matrix: ${{ fromJson(needs.generate_matrix.outputs.specs) }}
     steps:
       - uses: actions/checkout@v4
+      - name: Download OpenAPI specs artifacts
```

Comment:
would just leave a comment here where the download path is...

Reply:
https://github.com/eclipse-tractusx/api-hub/pull/9/commits/670b630398757c05548af7a9f67b99e0fbc1f3de